docker_multiarch/arm.crosscompile.mk

```diff
@@ -159,4 +159,4 @@ USE_CPP_PACKAGE = 0
 # whether to use sframe integration. This requires build sframe
 # git@github.com:dato-code/SFrame.git
 # SFRAME_PATH = $(HOME)/SFrame
-# MXNET_PLUGINS += plugin/sframe/plugin.mk
\ No newline at end of file
+# MXNET_PLUGINS += plugin/sframe/plugin.mk
```

Comment:
Whitespace change, maybe revert

Reply:
Should be a whitespace at the end of files, but I can put this in another commit.